client/js/restaurantsAPI.js

```diff
@@ -55,7 +55,13 @@ var restaurantsAPI = (function() {
   /* Simplify the restaurant format using only useful info */
```

Comment:
Unify conventions, use `//` for one line comments and `/* */` for multi-line ones